hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/nio/IOVector.java

```diff
@@ -18,82 +18,78 @@
 
 import com.hazelcast.internal.tpc.iobuffer.IOBuffer;
 
-import java.io.IOException;
 import java.nio.ByteBuffer;
-import java.nio.channels.SocketChannel;
 import java.util.Queue;
 
 /**
- * Contains logic to do vectorized I/O (so instead of passing a single buffer, an array of buffer is passed to socket.write).
+ * Contains logic to do vectorized I/O (so instead of passing a single buffer, an array
+ * of buffer is passed to socket.write).
  */
 public final class IOVector {
 
     private static final int IOV_MAX = 1024;
 
     private final ByteBuffer[] array = new ByteBuffer[IOV_MAX];
     private final IOBuffer[] bufs = new IOBuffer[IOV_MAX];
-    private int size;
```

Comment:
Renamed 'size' the 'length' because that is the typical name of the field passed for vectorized I/O.